src/pages/guides/hybrid-plugins/index.md

```diff
@@ -0,0 +1,44 @@
+---
+keywords:
+  - UXP Hybrid
+  - Hybrid
+  - C++
+title: Adobe UXP Hybrid plugin
+description: A UXP Hybrid plugin is a UXP plugin combined with the power of C++ native libraries. 
+contributors:
+  - https://github.com/padmkris123
+---
+
+# UXP Hybrid plugin
+
+A UXP Hybrid plugin is a UXP plugin combined with the power of C++ native libraries. 
+
+Take advantage of more performant C++ code to enhance your plugins. For instance, you can expand the capabilities of your plugin to
+- Execute computation-intensive tasks
+- Integrate with existing external pipelines, processes, and executables
+- Make use of other popular libraries such as OpenCV
+
+In order to write a Hybrid plugin, you must be proficient in C++. Consider this to be an advanced topic, so make sure you have
+
+- Experience in writing [UXP plugins](../getting-started/creating-your-first-plugin/)
+- Experience with [UDT (UXP Developer tool)](../devtool/)
+
+## UXP Hybrid plugin vs Photoshop C++ plugins
+
+A UXP hybrid plugin should not be confused with the Photoshop C++ plugin [built using CSDK](../../ps_reference/media/cpp-pluginsdk/). In a Photoshop C++ plugin, the UI (JavaScript) and C++ code reside in two separate plugins. But a UXP hybrid plugin allows you to bundle all the code within a single .ccx package.
```

Comment:
... with the Photoshop ...  -> ... with a Photoshop ...

"In a Photoshop C++ plugin, the UI (JavaScript) and C++ code reside in two separate plugins. But a UXP hybrid plugin allows you to bundle all the code within a single .ccx package."

I think it would be better to leave this bit to the other page.  You can create UI within a C++ plugin, so it reads oddly to me.  Better to focus on the compiled differences perhaps:

A Photoshop C++ plugin is specific to Photoshop in that it has access to the APIs detailed in the SDK.  For example, a filter is one of the seven types.  As noted above, a UXP hybrid plugin will be either general in nature or specific to an SDK other than Photoshop.  

The key with hybrid is the ease of developing UI for the plugin with UXP.  To create UI for a Photoshop C++ plugin, one must build with C++ or create a wholly separate UXP plugin to pair with it.  A UXP hybrid plugin allows ...

Reply:
OK. I have removed "In a Photoshop C++ plugin, the UI (JavaScript) and C++ code reside in two separate plugins. But a UXP hybrid plugin allows you to bundle all the code within a single .ccx package."